### ISSUE TITLE
Validators as symbol

### ISSUE THREAD

**Issue by mscavnicky** [jamesbrooks/hash_validator#9]
Thanks for making this gem, it comes handy at multiple occasions. Did you consider allowing validators as symbols?

```
validations = {
  user: {
    first_name: :string,
    last_name: :string,
    age: :numeric,
    likes: :array
  }
}
```

Using symbols seems idiomatic (see http://stackoverflow.com/questions/16621073/when-to-use-symbols-instead-of-strings-in-ruby).

**Comment by jamesbrooks** [comment]
Thanks for this suggestion!

I've implemented this change in 98ca9dd91475f4c8f193a8136f65fe82a04e73a8, which as also been pushed as gem version 0.6.0

**Comment by mscavnicky** [comment]
Thanks a lot!
